pages/common/hadolint.md

```diff
@@ -0,0 +1,16 @@
+# hadolint
+
+> Lints Dockerfiles to help you build [best practice](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices) Docker images.
+> More information: https://github.com/hadolint/hadolint.
+
+- Lint a Dockerfile, printing any output to STDOUT:
+
+`hadolint {{Dockerfile}}`
+
+- Lint a Dockerfile ignoring specific rules and outputting JSON:
```

Comment:
I think you should split the format output from ignore rules. Which are the possible formats?

Reply:
The formats are `tty | json | checkstyle | codeclimate | codacy`
Not sure how these are typically displayed

Reply:
Not sure, but you used less than 8 examples. Maybe is better add an extra example

Reply:
Added another example